EnlightenMAUI.csproj

```diff
@@ -49,13 +49,13 @@
 
 	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net8.0-android34.0|AnyCPU'">
 	  <IsPublishable>False</IsPublishable>
-	  <ApplicationDisplayVersion>0.5.1</ApplicationDisplayVersion>
+	  <ApplicationDisplayVersion>0.5.0</ApplicationDisplayVersion>
```

Comment:
I disagree that we should go backwards in version when adding a feature. This should probably be 0.5.2, or maybe even 0.6.0.

Reply:
overlooked this one